anesthetic/weighted_pandas.py

```diff
@@ -266,7 +266,7 @@ def sem(self, axis=0, skipna=True):
         n = self.neff() if axis == 0 else self.shape[1]
         return np.sqrt(self.var(axis=axis, skipna=skipna)/n)
 
-    def quantile(self, q=0.5, axis=0):
+    def quantile(self, q=0.5, axis=0, **kwargs):
```

Comment:
I've put in the generic `**kwargs` here. The actual `pandas.DataFrame.quantile` signature is:

```python
    def quantile(
        self,
        q=0.5,
        axis: Axis = 0,
        numeric_only: bool = True,
        interpolation: str = "linear",
    ):
```

Would it be better to explicitly use `numeric_only` and `interpolation` here, rather than `**kwargs`?

Reply:
Yes. 03f74ef now does this, by implementing the interpolation kwarg, and throwing a `NotImplementedError` if one tries to pass `numeric_only=False`

Reply:
I think that this is preferable to the `**kwargs` solution, since the tests will continue to pick up internal breaking changes like the one that triggered this.